ts/WoltLabSuite/Core/Acp/Ui/User/Content/Remove/Handler.ts

```diff
@@ -24,13 +34,15 @@ interface AjaxResponse {
 class AcpUserContentRemoveHandler {
   private readonly dialogId: string;
   private readonly userId: number;
+  private readonly callbackSuccess: CallbackSuccess | null | undefined;
 
   /**
    * Initializes the content remove handler.
    */
-  constructor(element: HTMLElement, userId: number) {
+  constructor(element: HTMLElement, userId: number, callbackSuccess?: CallbackSuccess | null) {
```

Comment:
```suggestion
  constructor(element: HTMLElement, userId: number, callbackSuccess: CallbackSuccess | null) {
```

Undefined is not a valid value for AcpUiWorker.